### PR TITLE
collapseable calendar

### PR DESCRIPTION
**Pull request by yashash1511**
# Description

image
image

## Type of change

- [x] New feature (non-breaking change which adds functionality)

### CODE REVIEW

**Comment by yashash1511** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by yashash1511** [comment]
recheck

**Comment by Posandu** [comment]
nice

**Comment by yashash1511** [comment]
it transitions fine while going from collapsed state to uncollapsed but not vice versa any guess why

File: src/components/start/sidepane.scss [hunk 1]
```diff
@@ -195,6 +195,11 @@ body[data-theme="dark"] .sidePane {
   z-index: 9999;
   border-radius: 8px;
 
+  &.collapse {
+    transform: translateY(calc(100% - 50px));
+    height: 50px;
```
Comment (Posandu) [inline src/components/start/sidepane.scss:200]: Change this to max height

File: src/components/start/sidepane.scss [hunk 2]
```diff
@@ -195,6 +195,11 @@ body[data-theme="dark"] .sidePane {
   z-index: 9999;
   border-radius: 8px;
```
Comment (Posandu) [inline src/components/start/sidepane.scss:197]: Add `max-height:500px` here

Comment (yashash1511) [reply src/components/start/sidepane.scss:197]: this pr has nothing to do with the sidepane

File: src/components/start/sidepane.scss (hunk 1 above)
Comment (yashash1511) [reply src/components/start/sidepane.scss:200]: tried still doesn't transition

File: src/components/start/sidepane.scss (hunk 2 above)
Comment (Posandu) [reply src/components/start/sidepane.scss:197]: oh wait

File: src/components/start/sidepane.scss (hunk 1 above)
Comment (yashash1511) [reply src/components/start/sidepane.scss:200]: ohhk got it

File: src/components/start/sidepane.scss (hunk 2 above)
Comment (yashash1511) [reply src/components/start/sidepane.scss:197]: yes

Comment (Posandu) [reply src/components/start/sidepane.scss:197]: apply this for `.calnpane`

Comment (yashash1511) [reply src/components/start/sidepane.scss:197]: thank you it works now

**Pull request merged**